packages/terra-collapse/tests/jest/Collapse.test.jsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import IconCaretRight from 'terra-icon/lib/icon/IconCaretRight';
+import Collapse from '../../src/Collapse';
+
+
+describe('Collapse', () => {
+  // Snapshot Tests
+  it('should render a default collapse', () => {
+    const collapse = shallow(<Collapse closedButtonText="Show">Test</Collapse>);
+    expect(collapse).toMatchSnapshot();
+  });
+
+  it('should render an animated collapse', () => {
+    const collapse = shallow(<Collapse closedButtonText="Show" isAnimated>Test</Collapse>);
+    expect(collapse).toMatchSnapshot();
+  });
+
+  it('should render an initially open collapse', () => {
+    const collapse = shallow(<Collapse closedButtonText="Show" isInitiallyOpen>Test</Collapse>);
+    expect(collapse).toMatchSnapshot();
+  });
+
+  it('should mount a collapse', () => {
+    const collapse = mount(<Collapse closedButtonText="Show">Test</Collapse>);
+    expect(collapse).toMatchSnapshot();
+  });
+
+  it('should render a collapse', () => {
```

Comment:
Is there something different about this snapshot test compared to the test on line 6?